tools/install_prereqs.sh

```diff
@@ -53,6 +53,7 @@ else
     python3-pip \
     python3-venv \
     rocblas-dev \
+    hipblaslt \
```

Comment:
dont we need to install the dev package?

Reply:
Updated.